decidim-core/lib/decidim/exporters/pdf.rb

```diff
@@ -15,42 +15,28 @@ class PDF < Exporter
       #
       # Returns an ExportData instance.
       def export
-        html = controller.render_to_string(
-          template:,
-          layout:,
-          locals:
-        )
+        composer.styles(**styles)
 
-        document = WickedPdf.new.pdf_from_string(html, orientation:)
+        add_data!
 
-        ExportData.new(document, "pdf")
+        ExportData.new(composer.write_to_string, "pdf")
       end
 
-      # may be overwritten if needed
-      def orientation
-        "Portrait"
-      end
+      protected
 
-      # implementing classes should return a valid ERB path here
-      def template
-        raise NotImplementedError
-      end
+      def layout = composer.document.layout
 
-      # implementing classes should return a valid ERB path here
-      def layout
-        raise NotImplementedError
+      def composer
+        @composer ||= ::HexaPDF::Composer.new
       end
 
-      # This method may be overwritten if the template needs more local variables
-      def locals
-        { collection: }
+      def add_data!
+        raise NotImplementedError
       end
 
-      protected
+      def font_family = "Times"
```

Comment:
Can you change this to a font with Sans? That's what it looked like until now

Reply:
I am not sure that Hexpdf has Sans. These are the fonts supported: 

- https://hexapdf.gettalong.org/examples/standard_pdf_fonts.html
- https://hexapdf.gettalong.org/examples/truetype.html

Reply:
Ok, since we already have Source Sans in TTF format, i have injected into the PDF. Can U check now ?